test/integration/etcd_storage_path_test.go

```diff
@@ -129,24 +130,77 @@ var etcdStorageData = map[reflect.Type]struct {
-	reflect.TypeOf(&authorizationapiv1.LocalResourceAccessReview{}):     {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.Role{}):                          {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.RoleBindingRestriction{}):        {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.Policy{}):                        {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.ClusterPolicyBinding{}):          {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.SubjectAccessReviewResponse{}):   {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.ClusterRole{}):                   {ephemeral: true}, // TODO(mo): Just making the test pass
-	reflect.TypeOf(&authorizationapiv1.SubjectRulesReview{}):            {ephemeral: true}, // TODO(mo): Just making the test pass
+	reflect.TypeOf(&authorizationapiv1.PolicyBinding{}): {
+		stub: &authorizationapiv1.PolicyBinding{
+			RoleBindings: authorizationapiv1.NamedRoleBindings{
+				{
+					Name: "rb",
+					RoleBinding: authorizationapiv1.RoleBinding{
+						ObjectMeta: kapiv1.ObjectMeta{Name: "rb", Namespace: testNamespace},
+						RoleRef:    kapiv1.ObjectReference{Name: "r"},
+					},
+				},
+			},
+		},
+		expectedEtcdPath: "openshift.io/authorization/local/policybindings/etcdstoragepathtestnamespace/:default",
+	},
+	reflect.TypeOf(&authorizationapiv1.ClusterPolicyBinding{}): {
+		stub: &authorizationapiv1.ClusterPolicyBinding{
+			ObjectMeta: kapiv1.ObjectMeta{Name: "objectisincomparewhitelist"},
+		},
+		expectedEtcdPath: "openshift.io/authorization/cluster/policybindings/:default",
+	},
+	reflect.TypeOf(&authorizationapiv1.Policy{}): {
+		stub: &authorizationapiv1.Policy{
+			Roles: authorizationapiv1.NamedRoles{
+				{
+					Name: "r",
+					Role: authorizationapiv1.Role{
+						ObjectMeta: kapiv1.ObjectMeta{Name: "r", Namespace: testNamespace},
+					},
+				},
+			},
+		},
+		expectedEtcdPath: "openshift.io/authorization/local/policies/etcdstoragepathtestnamespace/default",
+	},
+	reflect.TypeOf(&authorizationapiv1.ClusterPolicy{}): {
+		stub: &authorizationapiv1.ClusterPolicy{
+			ObjectMeta: kapiv1.ObjectMeta{Name: "objectisincomparewhitelist"},
+		},
+		expectedEtcdPath: "openshift.io/authorization/cluster/policies/default",
+	},
+	reflect.TypeOf(&authorizationapiv1.RoleBindingRestriction{}): {
+		stub: &authorizationapiv1.RoleBindingRestriction{
+			ObjectMeta: kapiv1.ObjectMeta{Name: "rbr"},
+			Spec: authorizationapiv1.RoleBindingRestrictionSpec{
+				ServiceAccountRestriction: &authorizationapiv1.ServiceAccountRestriction{
+					ServiceAccounts: []authorizationapiv1.ServiceAccountReference{
+						{
+							Name: "sa",
+						},
+					},
+				},
+			},
+		},
+		expectedEtcdPath: "openshift.io/rolebindingrestrictions/rbr",
```

Comment:
https://github.com/enj/origin/blob/master/pkg/authorization/registry/rolebindingrestriction/strategy.go#L25 does not agree with https://github.com/enj/origin/blob/master/pkg/authorization/registry/rolebindingrestriction/etcd.go#L46